test/EVMHost.h

```diff
@@ -68,6 +68,12 @@ class EVMHost: public evmc::Host
 		std::vector<LogEntry> logs;
 	};
 
+	Account const* account(evmc::address const& _address) const
+	{
+		auto it = m_state.accounts.find(_address);
+		return it == m_state.accounts.end() ? nullptr : &it->second;
+	}
+
 	Account* account(evmc::address const& _address)
 	{
 		// Make all precompiled contracts exist.
```

Comment:
This is now undermined by the ``const`` version above, isn't it?

Reply:
Right, it will change the `account_exists()` (which is const). I thought this is for calls. This should be moved to `account_exists()`. The rest of the methods should work the same.